matsim/src/main/java/org/matsim/api/core/v01/events/LinkLeaveEvent.java

```diff
@@ -35,31 +35,32 @@
 	
 	private final Id<Link> linkId;
 	private final Id<Vehicle> vehicleId;
+	private final Id<Person> driverId;
```

Comment:
The same about `driverId` here as in the `LinkEnterEvent`.

Reply:
Same as `LinkEnterEvent`.